types/sdk.js

```diff
@@ -124,6 +124,9 @@ export type ServiceBuilderDefinition = {
 export type ApiRequestBuilder = {
   [key: string]: ServiceBuilder;
 }
+export type BuildOptions = {
```

Comment:
Didn't want to really mess with the variable names, and since [you feel the variable should be called ```options```](https://github.com/commercetools/nodejs/pull/116#discussion_r108428216), what better could there be?